recipes/mysql-connector-cpp/all/conanfile.py

```diff
@@ -0,0 +1,49 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake
+
+
+class MysqlConnectorCppConan(ConanFile):
+    name = "mysql-connector-cpp"
+    version = "8.0.27"
+    license = "GPLv2"
+    author = "Oracle"
+    url = "https://dev.mysql.com/doc/connector-cpp/en/"
+    description = "MySQL Connector/C++ is a MySQL database connector for C++."
+    topics = ("mysql", "database", "connector", "c++")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
```

Comment:
the options for libraries in Conan Center should also include `fPIC`, as per:
https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py#L32-L39